src/control/server/config/server.go

```diff
@@ -326,6 +334,8 @@ func (cfg *Server) WithTelemetryPort(port int) *Server {
 // DefaultServer creates a new instance of configuration struct
 // populated with defaults.
 func DefaultServer() *Server {
+	clientFirewallMode := false
```

Comment:
There isn't anything necessarily wrong with doing it this way but in other areas where we are trying to do something similar this is set during validation e.g. VMD related settings.